### DISCUSSION TITLE
Is it possible to use Biome within a Monorepo?

### DISCUSSION THREAD

**Discussion by matthiashermsen** [biomejs/biome#1727] [Q&A] [answered]
I'm playing around with Turborepo / Nx and created a package with the name *@repo/biome-configuration* containing multiple .json files.

All projects have a biome.json file with the content

```json
{
	"extends": ["@repo/biome-configuration/TEMPLATE_GOES_HERE.json"]
}
```

It seems this doesn't work I guess. Biome is not able to find a file like `"@repo/biome-configuration/react.json"`.

Are there any solutions for this?

**Comment by ematipico** [comment, marked as answer]
Please, read the documentation

https://biomejs.dev/reference/configuration/#extends

____

The files defined in this array:

    must exist in the file system;
    are resolved from the path where the biome.json file is defined;
    must be relative paths. Paths to libraries are not resolved;
    must be reachable by Biome, e.g. symbolic links might not be resolved by Biome;
    will be processed in order: from the first one to the last one;
    can override the same properties, but ultimately only the last one will be used by Biome;
